report_thesis/src/sections/background/linear_and_regularization_models/lasso.tex

```diff
@@ -1,7 +1,11 @@
 \subsubsection{Least Absolute Shrinkage and Selection Operator (LASSO) Regression}
-Having introduced ridge regression, we now turn our attention to the \gls{lasso} regression based on \citet{James2023AnIS}.
+A disadvantage of ridge regression is that it includes all features in the final model.
+While it does shrink the coefficients towards zero, it does not eliminate any.
```

Comment:
Should we clarify that the `elimination` here refers to setting the weight for that feature to zero? Or is it fine to imply like this. Feature obv. doesn't get deleted.

Reply:
```suggestion
While it does shrink the coefficients towards zero, it does not set any to exactly zero.
```